### PR TITLE
Add brick_utilization alert handler

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by r0h4n** [comment]
Verified?

File: tendrl/alerting/handlers/cluster/brick_utilization_handler.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from tendrl.alerting.objects.cluster_alert import ClusterAlert
+
+
+class BrickUtilizationHandler(AlertHandler):
+
+    handles = 'brick_utilization'
+    representive_name = 'brick_utilization_alert'
+
+    def __init__(self):
+        AlertHandler.__init__(self)
```
Comment (shtripat) [inline tendrl/alerting/handlers/cluster/brick_utilization_handler.py]: `super(BrickUtilizationHandler, self).__init__()` looks more clear to me.

Comment (anmolbabu) [inline tendrl/alerting/handlers/cluster/brick_utilization_handler.py]: May 26 04:11:30 dhcpxx-xxx tendrl-alerting: super(BrickUtilizationHandler, self).__init__()
May 26 04:11:30 dhcpxx-xxx tendrl-alerting: NameError: global name 'BrickUtilizationHandler' is not defined

**Comment by anmolbabu** [comment]
Verified

**Pull request merged**